macros/anaUltraScurve.py

```diff
@@ -1,4 +1,4 @@
-#!/bin/env python
+#!/bin/env python2.7
 import os
```

Comment:
Hmm, it's working correctly at p5 which has a default ```slc6``` (i.e., non-modern ```python```), definitely seems like an env error somewhere